scripts/start-database.sh

```diff
@@ -17,6 +16,13 @@ echo " - $migrations_image"
 docker pull $db_image
 docker pull $migrations_image
 
+# If the database is already running, stop it
+if docker top $NAME &>/dev/null; then
+    echo "Stopping database"
+    docker stop $NAME
+    sleep 1s
+fi
```

Comment:
This isn't specific to this PR; it's just a handy thing I've been meaning to add for a while.

Reply:
will this not error if the db container isn't running?

Reply:
lol sorry github just showed me lines 21-23, I see that you are checking for that!